enhancements/installer/aws-permissions-check-bypass.md

```diff
@@ -0,0 +1,318 @@
+---
+title: aws-permissions-check-bypass
+authors:
+  - "@joelddiaz"
+reviewers:
+  - "@dgoodwin"
+  - "@abhinavdahiya"
+  - "@jeremyeder"
+approvers:
+  - TBD
+creation-date: 2020-05-07
+last-updated: 2020-05-15
+status: provisional
+see-also:
+  - ""  
+replaces:
+  - ""
+superseded-by:
+  - ""
+---
+
+# AWS Permissions Check Bypass
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions [optional]
+
+## Summary
+
+Running the OpenShift installer in AWS accounts using Service Control Policies
+(SCP) can result in errors indicating that the credentials provided do not have
+sufficient permissions when in reality they do. This is a limitation with the
+AWS IAM permissions simulation API.  Additionally, even without these
+limitations it is also true that the AWS policy language is sophisticated enough
+to restrict permissions in a way where is is difficult to perform acurate policy
+simulations (eg restricting permissions based on VPC or source IP).
```

Comment:
As an outsider, I know there is a problem with SCP's and installing.  In this summary, it states it's a problem with the "AWS IAM permissions simulation".   

Can this be expounded a bit to explain what the actual problem is?

Reply:
it looks like this is covered more in the "motivation section"